Demo/Sources/Fullscreen/Stories/StoryDemoView.swift

```diff
@@ -3,7 +3,13 @@ import UIKit
 import FinnUI
 
 class StoryDemoView: UIView {
-    private lazy var storiesView: StoriesView = StoriesView(dataSource: self, delegate: self, withAutoLayout: true)
+    private lazy var storiesView: StoriesView = StoriesView(
+        dataSource: self,
+        delegate: self,
+        errorViewModel: errorViewModel,
+        withAutoLayout: true
+    )
+    private let errorViewModel: StoryErrorViewModel = .init(title: "Usjda", description: "Noe gikk galt")
```

Comment:
Localized somewhere after?

Reply:
yes, this is demo code only. It's localized in the app 👍